Makefile

```diff
@@ -34,7 +34,7 @@ clean-env:
 # before "install" target.
 board-configs:
 	@echo "Note: you should have *all* of your boards connected to USB before running this."
-	@echo "Press Ctrl+C to break if not. Review results carefully afterwards."
+	@echo "Press any key when you're done or press Ctrl+C to break if not. Review results carefully afterwards."
```

Comment:
I guess to avoid any confusion, this then should say something like "Press Ctrl+C to break if not, or Enter to continue."

Reply:
Ok, I was not completely satisfied with my proposal. I will update, txs.